app.js

```diff
@@ -28,14 +28,15 @@ const app = express();
 // database cleanup task
 // removes actions and completed requests older than 60 days
 async function cleanupDb() {
+  const expiry = process.env.DATABASE_EXPIRY ? process.env.DATABASE_EXPIRY : 999;
```

Comment:
Need to `parseInt` here and shouldn't we set the default to 60 to match old behaviour?

Reply:
Yes.